packages/grafana-ui/src/components/TransformersUI/CalculateFieldTransformerEditor.tsx

```diff
@@ -24,14 +50,21 @@ interface CalculateFieldTransformerEditorState {
   selected: string[];
 }
 
+const calculationModes = [
+  { value: CalculateFieldMode.ReduceRow, label: 'Reduce Row' },
+  { value: CalculateFieldMode.BinaryOperation, label: 'Binary Operation' },
+  { value: CalculateFieldMode.Scale, label: 'Scale Field' },
```

Comment:
Please rename to Reduce row (lowercase r) and the rest. We are trying to align all buttons and headings to use sentence case.